// @slidev/cli/dist/index.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});










var _chunkT7KA74KVjs = require('./chunk-T7KA74KV.js');



var _chunk2IBHRHAWjs = require('./chunk-2IBHRHAW.js');


var _chunkN6L4BUFDjs = require('./chunk-N6L4BUFD.js');

// node/index.ts
_chunkN6L4BUFDjs.init_cjs_shims.call(void 0, );

// node/declare.ts
_chunkN6L4BUFDjs.init_cjs_shims.call(void 0, );













exports.ViteSlidevPlugin = _chunk2IBHRHAWjs.ViteSlidevPlugin; exports.createServer = _chunkT7KA74KVjs.createServer; exports.createWindiCSSPlugin = _chunk2IBHRHAWjs.createWindiCSSPlugin; exports.getAddonRoots = _chunkT7KA74KVjs.getAddonRoots; exports.getCLIRoot = _chunkT7KA74KVjs.getCLIRoot; exports.getClientRoot = _chunkT7KA74KVjs.getClientRoot; exports.getRoot = _chunkT7KA74KVjs.getRoot; exports.getThemeRoots = _chunkT7KA74KVjs.getThemeRoots; exports.getUserRoot = _chunkT7KA74KVjs.getUserRoot; exports.isPath = _chunkT7KA74KVjs.isPath; exports.parser = _chunkT7KA74KVjs.parser; exports.resolveOptions = _chunkT7KA74KVjs.resolveOptions;
